Firestore/core/CMakeLists.txt

```diff
@@ -206,7 +206,8 @@ firebase_ios_glob(
   src/objc/*.h
   src/remote/*.cc
   src/remote/*.h
-  src/remote/grpc_adapt/grpc_adaption.h
+  src/remote/grpc_adapt/*.h
```

Comment:
CMake should not include these. They should only be there to support grpc-swift.

Reply:
Removed